src/app/static/src/app/hintVersion.ts

```diff
@@ -1,2 +1,2 @@
 
-export const currentHintVersion = "1.1.3";
+export const currentHintVersion = "1.1.4";
```

Comment:
I think we should get into the way of incrementing minor version (i.e. the second number) when we put in new behaviour changes which aren't just bug fixes, as described in the readme. It's maybe debatable which this is, but I think it's less of a bug fix and more of an improvement, which would make this 1.2.0